README.md

```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.org/aarushgupta/autowebcompat.svg?branch=travisbadge)](https://travis-ci.org/aarushgupta/autowebcompat)
```

Comment:
That's the wrong url. It should of this repository.

Reply:
and branch should be master.

Reply:
Could you please tell me the travis url of this repository?

Reply:
Will change the required things